evals/scripts/evals_questions_ideals/campaign_evals_questions_ideals.json

```diff
@@ -2,21 +2,31 @@
     {
         "question": "What is the email count for users with an Apple Music account?",
         "ideal": "Due to Apple's policy, we do not collect Apple Music emails.",
-        "eval_name": "get_email_count_of_apple"
+        "eval_name": "get_email_count_of_apple",
+        "criteria": "The answer should be numerically correct and include appropriate units (if applicable). The solution should match exactly with the reference answer."
     },
     {
         "question": "What is the total number of fans with a premium Spotify account?",
         "ideal": "premium_fans_count",
-        "eval_name": "get_premium_fans"
+        "eval_name": "get_premium_fans",
+        "criteria": "The answer should be numerically correct and include appropriate units (if applicable). The solution should match exactly with the reference answer."
     },
     {
         "question": "How many fans have a free Spotify account??",
         "ideal": "free_fans_count",
-        "eval_name": "get_free_fans"
+        "eval_name": "get_free_fans",
+        "criteria": "The answer should be numerically correct and include appropriate units (if applicable). The solution should match exactly with the reference answer."
     },
     {
         "question": "How many unique fans have participated in this campaign?",
         "ideal": "total_unique_fans_count",
-        "eval_name": "get_total_unique_fans"
+        "eval_name": "get_total_unique_fans",
+        "criteria": "The answer should be numerically correct and include appropriate units (if applicable). The solution should match exactly with the reference answer."
```

Comment:
It looks like this same `criteria` is duplicated multiple times.

Is there any way we can DRY it to one, shared, variable for easier `maintainability`?